core/dbt/include/global_project/macros/materializations/test.sql

```diff
@@ -1,3 +1,19 @@
+{% macro test_body(main_sql, fail_calc, warn_if, error_if, limit) -%}
```

Comment:
Let's name this macro something that doesn't start with prefix `test_`, since that's the prior indicator of a test-definition macro. How about: `get_test_sql`?

Reply:
done!